pkg/controller/identity/iamrole/controller_test.go

```diff
@@ -57,7 +58,7 @@ func TestMain(m *testing.M) {
 func Test_Connect(t *testing.T) {
```

Comment:
Can we refactor these tests to match our table-driven pattern in other `v1beta1` resources?

Ref: https://github.com/crossplane/stack-aws/blob/94baed407fc79f3a083c409b42542a00f7f0eb74/pkg/controller/database/rdsinstance_test.go#L106